source/unit/Fixture.ooc

```diff
@@ -170,9 +170,10 @@ Fixture: abstract class {
 }
 
 TestFailedException: class extends Exception {
+	description: String
```

Comment:
Wasn't it safe to run `free` multiple times on string literals ? Maybe it was a dream.